### ISSUE TITLE
get_quantity unit tests fail on R 4.0 devel

### ISSUE THREAD

**Issue by adokter** [adokter/bioRad#377]
The following lines do not pass:

https://github.com/adokter/bioRad/blob/e0b3f3b690e67ebfdde319d6afb93da1b3eff9aa/tests/testthat/test-get_quantity.R#L103-L120

`rhub::check_with_rdevel()` gives:

```  
check_with_rdevel()
─  Building package
─  Uploading package
─  Preparing build, see status at
   https://builder.r-hub.io/status/bioRad_0.5.1.9344.tar.gz-0f14ac1d66f4425fbf522f288feeda02
─  Build started
─  Downloading and unpacking package file
─  Querying system requirements
─  Installing system requirements
─  Starting Docker container
─  Querying package dependencies
─  Installing package dependencies
─  Running R CMD check
   About to run xvfb-run R CMD check bioRad_0.5.1.9344.tar.gz
─  using log directory ‘/home/docker/bioRad.Rcheck’
─  using R Under development (unstable) (2020-04-26 r78302)
─  using platform: x86_64-pc-linux-gnu (64-bit)
─  using session charset: UTF-8
✓  checking for file ‘bioRad/DESCRIPTION’
─  this is package ‘bioRad’ version ‘0.5.1.9344’
─  package encoding: UTF-8
✓  checking package namespace information
✓  checking package dependencies (2.6s)
✓  checking if this is a source package
✓  checking if there is a namespace
✓  checking for executable files (2.6s)
✓  checking for hidden files and directories
✓  checking for portable file names
✓  checking for sufficient/correct file permissions
✓  checking whether package ‘bioRad’ can be installed (21.3s)
N  checking installed package size
     installed size is  5.7Mb
     sub-directories of 1Mb or more:
       data      1.6Mb
       doc       1.2Mb
       extdata   1.7Mb
✓  checking package directory
✓  checking ‘build’ directory
✓  checking DESCRIPTION meta-information (2.7s)
✓  checking top-level files
✓  checking for left-over files
✓  checking index information
✓  checking package subdirectories
✓  checking R files for non-ASCII characters
✓  checking R files for syntax errors
✓  checking whether the package can be loaded (5.2s)
✓  checking whether the package can be loaded with stated dependencies (2.9s)
✓  checking whether the package can be unloaded cleanly (6s)
✓  checking whether the namespace can be loaded with stated dependencies (2.6s)
✓  checking whether the namespace can be unloaded cleanly (5.6s)
✓  checking loading without being on the library search path (2.8s)
✓  checking dependencies in R code (5.6s)
✓  checking S3 generic/method consistency (7.6s)
✓  checking replacement functions (5.2s)
✓  checking foreign function calls (5.2s)
✓  checking R code for possible problems (21.4s)
✓  checking Rd files
✓  checking Rd metadata
✓  checking Rd cross-references (2.5s)
✓  checking for missing documentation entries (2.8s)
✓  checking for code/documentation mismatches (13.8s)
✓  checking Rd \usage sections (7.7s)
✓  checking Rd contents
✓  checking for unstated dependencies in examples (2.7s)
✓  checking contents of ‘data’ directory
✓  checking data for non-ASCII characters
✓  checking data for ASCII and uncompressed saves
✓  checking installed files from ‘inst/doc’
✓  checking files in ‘vignettes’
✓  checking examples (38.2s)
✓  checking for unstated dependencies in ‘tests’
─  checking tests
E  Running ‘testthat.R’ (46.7s)
   Running the tests in ‘tests/testthat.R’ failed.
   Last 13 lines of output:
     ══ testthat results  ═══════════════════════════════════════════════════════════
     [ OK: 282 | SKIPPED: 38 | WARNINGS: 4 | FAILED: 17 ]
     1. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#104) 
     2. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#105) 
     3. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#106) 
     4. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#107) 
     5. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#108) 
     6. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#109) 
     7. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#110) 
     8. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#111) 
     9. Failure: get_quantity.vpts() return a matrix for all 16 quantities (@test-get_quantity.R#112) 
     1. ...
     
     Error: testthat unit tests failed
     Execution halted
✓  checking for unstated dependencies in vignettes
✓  checking package vignettes in ‘inst/doc’
─  checking running R code from vignettes
     ‘bioRad.Rmd’ using ‘UTF-8’... OK
     ‘rad_aero_19.Rmd’ using ‘UTF-8’... OK
     ‘range_correction.Rmd’ using ‘UTF-8’... OK
    NONE
✓  checking re-building of vignette outputs (45.7s)
✓  checking PDF version of manual (5.1s)
   
─  Done with R CMD check
─  Saving artifacts
```

**Comment by peterdesmet** [comment]
Seems to be caused by https://github.com/tidymodels/recipes/issues/422, where `class()` is now returning a vector with 2 instead of 1 value: `"matrix" "array"`

We could do:

```R
expect_equal(class(get_quantity(vpts, "height"))[1], "matrix")
```

But the [referenced page](https://developer.r-project.org/Blog/public/2019/11/09/when-you-think-class.-think-again/index.html) suggest to use `is()` and no longer `class()`. So:

```R
expect_true(is(get_quantity(vpts, "height"), "matrix"))
```

**Comment by peterdesmet** [comment]
See #379.

**Comment by adokter** [comment]
closed by #379
